fjohnson_assignment_4/classy_tiny_instacart_2.rb

```diff
@@ -0,0 +1,188 @@
+module NewUser
+
+    class << self
+       attr_accessor :name, :phone, :address
+    end
+
+    def name
+        @name
+    end
+
+    def phone
+        @phone
+    end
+    
+    def address
+        @address
+    end
+
+    def create_account
+        puts "Welcome to Instacart!"
+        print "\nWhat is your name? "
+        @name = gets.chomp
+        print "Hi, #{@name}! What is your phone number? "
+        @phone = gets.chomp
+        print "What is your address? "
+        @address = gets.chomp
+        puts "\nThank you! Your account has been created."
+        puts "\n"
+        puts "_" * 50
+        puts "\tName: #{@name}"
+        puts "\tPhone Number: #{@phone}"
+        puts "\tAddress: #{@address}"
+        puts "-" * 50
+    end
+end
+
+module GroceryList
+
+    class << self
+        attr_accessor :item_name, :item_brand, :item_quantity, :item_instructions
+    end
+
+    def item_name
+        @item_name
+    end
+
+    def item_brand
+        @item_brand
+    end
+
+    def item_quantity
+        @item_quantity
+    end
+
+    def item_instructions
+        @item_instructions
+    end
+
+    def create_item
+        print "\nWhat item would you like to add? "
+        @item_name = gets.chomp
+        print "What brand? "
+        @item_brand = gets.chomp
+        print "How many? "
+        @item_quantity = gets.chomp.to_i
+        print "Do you have additional instuctions? Choose Y for yes or N for no. "
+        answer = gets.chomp
+        if answer == "N"
+            @item_instructions = ""
+        else answer == "Y"
+            print "What are your additional instructions? "
+            @item_instructions = gets.chomp
+        end
+    end
+
+    def add(attributes)
+        $grocery_list ||= []
+        item = new(attributes)
+        $grocery_list.push(item)
+        sleep 1
+        puts "\nAdding #{item.attributes[:name]} to your order..."
+        puts "Brand: #{item.attributes[:brand]}"
+        puts "Quantity: #{item.attributes[:quantity]}"
+        puts "Instructions: #{item.attributes[:instructions]}"
+        sleep 1
+    end
+
+    def add_more
+        print "\nWould you like to add another item? Choose Y for yes or N for no. "
+        answer = gets.chomp
+        if answer == "N"
+            puts "Okay! Finished adding items."
+        else answer == "Y"
```

Comment:
This doesn't really do anything, if you wanted to confirm that the value was "Y", try using a case statement instead. https://www.rubyguides.com/2015/10/ruby-case/